### ISSUE TITLE
PHP 8.0 Dependencies

### ISSUE THREAD

**Issue by gravataLonga** [drupol/php-conventions#2]
To be fair, i'm not have any scenario to reproduce the problem, but i got my own library using your library on PHP8.0 but it keep fail because some internal dependencies on this package. 

You can see here: 
https://github.com/gravataLonga/container/runs/1570139750?check_suite_focus=true. 

Because PHP8.0 it's already release, i'm updating my own library.

**Comment by drupol** [comment]
Hello there,

Indeed some dependencies are not yet available for PHP 8 :(

I don't know yet what's the best way to deal with that, besides waiting or removing them.

Got any idea ?

**Comment by drupol** [comment]
Ok,

I published version `^3`, could you please let me know how it goes?

**Comment by gravataLonga** [comment]
@drupol hi, sorry for later reply. Will try test now. :)

**Comment by gravataLonga** [comment]
It's strange.. it's said that `3.0` don't exist.. but i clearly can see that you release a new tag. 

```
Root composer.json requires drupol/php-conventions ^3.0, found drupol/php-conventions[dev-master, 1.7.4, ..., 1.8.19, 2.0.0, ..., 2.0.4] but it does not match the constraint.

```

**Comment by drupol** [comment]
I manually refresh the packagist page... and it should work now: https://packagist.org/packages/drupol/php-conventions

**Comment by gravataLonga** [comment]
Wow, that was fast. Thank it's installing, let me try again with PHP 8.0 if pass. :)

**Comment by drupol** [comment]
Is it working good?

**Comment by gravataLonga** [comment]
I'm having hard time to make work. I'm thinking that the problem isn't your library and because of that I didn't reply yet.

**Comment by drupol** [comment]
Okay !

**Comment by gravataLonga** [comment]
@drupol sorry for delay, can confirm it work on PHP8.0. Can close issue. :) Thanks for your time.

**Comment by drupol** [comment]
Great !
